images/ceph/Makefile

```diff
@@ -28,6 +28,7 @@ TEMP := $(shell mktemp -d)
 # Build Rook
 
 do.build:
+	@echo $(CEPH_IMAGE) > $(OUTPUT_DIR)/CEPH_IMAGE
```

Comment:
instead of saving this image tag, we have an example in minikube.sh, which does the following:
- `source "${scriptdir}/../../build/common.sh"`
- The `${BUILD_REGISTRY}` env var has the name of the built image, then we can tag it as `rook/ceph:master`